app/src/main/java/com/example/posebook/MapsFragment.kt

```diff
@@ -43,5 +61,37 @@ class MapsFragment : Fragment() {
         super.onViewCreated(view, savedInstanceState)
         val mapFragment = childFragmentManager.findFragmentById(R.id.map) as SupportMapFragment?
         mapFragment?.getMapAsync(callback)
+
+
+                if(1>0){
+                    showMapReviewPopup()
+                }
+        //this part is for comment pop up
+//        mapPopTest.setOnClickListener {
+//            val window = PopupWindow(this)
+//            val view = layoutInflater.inflate(R.layout.fragment_maps,null)
+//            window.contentView = view
+//            val imageView = view.findViewById<TextView>(R.id.mapBackgroundBox)
+//            imageView.setOnClickListener {
+//                window.dismiss()
+//            }
+//            window.showAsDropDown(mapPopTest)
+//        }
```

Comment:
Just add a `TODO` and call the `showMapReviewPop()` maybe when the default marker gets clicked. Remove any commented code that isn't needed.